samples/user-service/generated-go/user_service.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import "github.com/nexus-rpc/sdk-go/nexus"
+
+// A service for managing users.
+var UserService = struct {
```

Comment:
For Go it would be great if we had a way to ensure that a handler implements all operations in the service.

See what I generated in the prototype: https://github.com/bergundy/nexus-idl-prototype/blob/e8aa3ef744884ebc3d56677fff54ba1919c30af0/src/plugins/temporal/samples/go/gen/gen.go#L47-L79